environments.json

```diff
@@ -1513,6 +1515,13 @@
     "note_id": "experimental-flag",
     "unstable": true
   },
+  "chrome64": {
+    "full": "Chrome Canary",
```

Comment:
If you're adding this entry then Chrome 63 should be marked as `Chrome Dev` and Chrome 62 as `Chrome Beta`. In its current state the PR makes two different Canary versions to co-exist which is confusing.

Reply:
Chrome 62 is stable, not beta

Reply:
Fixed according to OmahaProxy
https://www.chromium.org/developers/calendar
(both stable and beta are version 62 there so I omit beta for now)

Reply:
True, there should be no version marked as beta currently in that case.